### ISSUE TITLE
Invoking papi counters give segfault

### ISSUE THREAD

**Issue by pagrubel** [STEllAR-GROUP/hpx#1890]
```
bin/1d_stencil_4 --nx=1000 --np=1000 -t4 \
    --hpx:print-counter /arithmetics/add@/papi{locality#0/worker-thread#*}/PAPI_TOT_CYC
Segmentation fault
```
I tried this with current master and the 09.11 tag

**Comment by hkaiser** [comment]
@maciekab Is that something you would have time to look into?

**Comment by pagrubel** [comment]
I'm doing a bisect see if I can narrow this

**Comment by pagrubel** [comment]
also segfaults when papi counter is not used with statistics
bin/1d_stencil_4 --nx=1000 --np=1000 -t4 --hpx:print-counter /papi{locality#0/worker-thread#*}/PAPI_TOT_CYC
Segmentation fault
-----------------------------------
and for hello_world
bin/hello_world --np=1000 -t4 -:print-counter /papi{locality#0/worker-thread#*}/PAPI_TOT_CYC
Segmentation fault

**Comment by pagrubel** [comment]
ecb63d4b1fd125abf4b5f812aa6db4dfa131a326 is the first bad commit

**Comment by hkaiser** [comment]
>  ecb63d4  is the first bad commit

@pagrubel Ahh thanks, this makes sense. I will fix things.

**Comment by pagrubel** [comment]
tried it and now get this:
unknown counter type: /papi/PAPI_TOT_CYC,

**Comment by pagrubel** [comment]
tried fixing_1890 and now I get this:
 unknown counter type: /papi/PAPI_TOT_CYC,

**Comment by hkaiser** [comment]
Please try again, I added the missing `#include` directives

**Comment by pagrubel** [comment]
I had to make one more change and now it works.

**Comment by hkaiser** [comment]
This was fixed by merging #1893
